### DISCUSSION TITLE
Force light map in iOS?

### DISCUSSION THREAD

**Discussion by gogoSpace** [react-native-maps/react-native-maps#4958] [Q&A] [answered]
In Swift, I can achieve the following:

```
if #available(iOS 13.0, *) {
    self.overrideUserInterfaceStyle = .light
}
```

This code is placed within the viewDidLoad() method of an MKMapView controller.

Now, I'm wondering how I can achieve a similar functionality in React Native. Specifically, I want to provide users with the option to force a light map even when the device is in dark mode.

**Comment by mateki0** [comment, marked as answer]
Hi @gogoSpace you can use userInterfaceStyle property to do that. Check out docs for more informations https://github.com/react-native-maps/react-native-maps/blob/master/docs/mapview.md

**Reply by gogoSpace** [reply]
Thank you.  It's a shame I missed this in the documentation.
